features/federated-credentials.yml

```
@@ -0,0 +1,11 @@
+name: Federated credentials
+description: The `FederatedCredential` API... not to be confused with FedCM.
```

I have literally no idea how this is different from federated credential _management_. 🤷 

```suggestion
description: The `FederatedCredential` API delegates authentication to a third-party identity provider directly, instead of depending on third-party cookie. Not to be confused with federated credential management (FedCM).
```

---

When I asked @nsatragno about the difference a few months ago she said:

> They're designed very differently and `FederatedCredential` has enough usage that I don't think it'll be deprecated any time soon. `FederatedCredential` only stores a set of usernames with their provider. FedCM is much more involved and has the browser mediate between the website and the identity provider. They solve the same problem in the same way a typewriter and an IDE solve the same problem of writing code...

But I'm not sure how to describe them in simple terms that gets at the difference. @nsatragno do you have any suggestions, starting from the description @ddbeck has suggested? Note that it's the same description as for FedCM in all substance.

---

Yeee this is very confusing and will not help developers u.u. But they are _very_ different things.

FederatedCredential is an API that relying parties (i.e. websites a user signs in to) can use to store and retrieve username and identity provider url pairs. This can be used to have the browser remember these details for the next time the user signs in. However, FedereatedCredential does not concern itself with how the federated sign-in works. It's a pretty basic storage, very similar to PasswordCredential.

The FedCM API instead defines a protocol used by identity providers (i.e. websites that can provide identity for a user, "sign in with...") to support sign-in flows mediated by the browser. The protocol defines endpoints that the identity provider must implement and will be accessed by the browser. The relying party calls the FedCM javascript API to trigger the browser showing federated authentication UI.

For more information on FedCM, you can read https://developers.google.com/privacy-sandbox/3pcd/fedcm-developer-guide.

--

I would describe FederatedCredential to developers as:

```suggestion
description: The `FederatedCredential` API allows websites to store the details required to authenticate a user using a third-party identity provider. Not to be confused with federated credential management (FedCM).
```

---

Thanks @nsatragno and @ddbeck! Applying the suggestion.